Lotto/Lotto/Lotto/View/InputView.swift

```diff
@@ -31,21 +34,32 @@ struct InputView {
     
     func recievePurchaseMoney() throws -> Int {
         userGuider.printGuide(for: QuestionText.purchaseMoney)
-        
-        let userInput: String? = readLine()
-        let unwrappedUserInput: String = try stringConverter.unwrapOptional(from: userInput)
-        let purchaseMoney = try userInputConverter.convertToMoney(from: unwrappedUserInput)
+        let purchaseMoney = try recieveInt()
         try purchaseMoneyValidator.validate(of: purchaseMoney)
         return purchaseMoney
     }
     
-    func recieveWinningNumbers() throws -> [Int] {
+    func recieveWinningLotto() throws -> Lotto {
```

Comment:
```suggestion
    func receiveWinningLotto() throws -> Lotto {
```
아래 메서드들도 확인 부탁드립니다 :)

Reply:
오잉 오타가 거하게 났군여

Reply:
https://github.com/next-step/swift-lotto/pull/21/commits/f23ade2ea554c2668971edbc16ac148b5eb5158b